crates/starknet-server/tests/get_transaction_receipt_by_hash.rs

```diff
@@ -148,7 +148,65 @@ mod get_transaction_receipt_by_hash_integration_tests {
     }
 
     #[tokio::test]
-    async fn get_declare_v1_transaction_receipt_by_hash_happy_path() {
```

Comment:
Do we now not have a happy_path case? Or was this test wrongly named?

Reply:
the happy_path is not happy, because it returns an error